src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/OpportunitiesOrchestrator.cs

```diff
@@ -186,5 +200,45 @@ private async Task<CreateApplicationCacheItem> RetrievePledgeCacheItem(Guid key)
 
             return result;
         }
+
+        public async Task<ApplicationDetailsViewModel> GetApplicationViewModel(ApplicationDetailsRequest request)
+        {
+            var application = await RetrieveCacheItem(request.CacheKey);
+            var applicationDetails = await _opportunitiesService.GetApplicationDetails(request.PledgeId);
+
+            return new ApplicationDetailsViewModel()
+            {
+                CacheKey = request.CacheKey,
+                EncodedAccountId = request.EncodedAccountId,
+                EncodedPledgeId = request.EncodedPledgeId,
+                JobRole = application.JobRole,
+                NumberOfApprentices = application.NumberOfApprentices,
+                Month = application.StartDate?.Month,
+                Year = application.StartDate?.Year,
+                HasTrainingProvider = application.HasTrainingProvider,
+                OpportunitySummaryViewModel = new OpportunitySummaryViewModel
+                {
+                    Description = GenerateDescription(applicationDetails.Opportunity, request.EncodedPledgeId),
+                    Amount = applicationDetails.Opportunity.Amount,
+                    JobRoleList = string.Join(", ", applicationDetails.Opportunity.JobRoles),
+                    LevelList = string.Join(", ", applicationDetails.Opportunity.Levels),
+                    SectorList = string.Join(", ", applicationDetails.Opportunity.Sectors),
+                    YearDescription = "2021/22"
```

Comment:
Is this YearDescription fine to be hardcoded? Seen this pop up in a few places.

Reply:
Yes, I believe we are OK to hard code it. Maybe we should hard code it in only once place though